processor/src/main/java/org/mapstruct/ap/internal/model/source/SubclassMappingOptions.java

```diff
@@ -31,9 +32,16 @@
 
     private final TypeMirror source;
     private final TypeMirror target;
+    private final FormattingMessager messager;
+    private final TypeUtils typeUtils;
+    private final SubclassMappingGem subclassMapping;
```

Comment:
Where is this used?

Reply:
inside isValidUsage

Reply:
got removed during fixing of previous remark.